src/interfaces/r/DESCRIPTION.in

```diff
@@ -0,0 +1,26 @@
+Package: shogun
+Version: @VERSION@
+Date: @DATE@
+Title: The SHOGUN Machine Learning Toolbox
+Author: Shogun Team
+Maintainer: Shogun Team <shogun-team@shogun-toolbox.org>
+Depends: R (>= 2.10.0)
+Suggests:
+Description: SHOGUN - is a new machine learning toolbox with focus on large
+    scale kernel methods and especially on Support Vector Machines (SVM) with focus
```

Comment:
while we're at it, could we update this to the one in the main readme / website?